tests/test_analog_signals.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import pytest
+
+from ophyd.sim import make_fake_device
+from pcdsdevices.analog_signals import Acromag
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def fake_acromag():
+    FakeAcromag = make_fake_device(Acromag)
+    acromag = FakeAcromag('Test:Acromag', name='test_acromag')
+    return acromag
+
```

Comment:
If we have methods that do things like set voltage, tweak voltage in this PR, they should also be tested in this PR.

Reply:
Yes definitely, I'll implement tests for the mesh functions next and update with another commit